go/flags/endtoend/mysqlctl.txt

```diff
@@ -62,9 +66,9 @@ Global flags:
       --log_rotate_max_size uint                                    size in bytes at which logs are rotated (glog.MaxSize) (default 1887436800)
       --logtostderr                                                 log to standard error instead of files
       --max-stack-size int                                          configure the maximum stack size in bytes (default 67108864)
-      --mysql_port int                                              MySQL port (default 3306)
+      --mysql_port int                                              MySQL port. (default 3306)
```

Comment:
Why do some flags have a period at the end of the help text and others don't?

Reply:
mistake (meant to update all of the local ones)

Reply:
oh i see what you're asking. these flags get defined local to `mysqlctl`, so i made them all (mostly, missed a few which i'm fixing now) proper sentences. the ones defined all over the codebase are bit less consistent